olp-cpp-sdk-core/include/olp/core/client/CancellationToken.h

```diff
@@ -26,30 +26,32 @@
 
 namespace olp {
 namespace client {
+
 /**
- * @brief Control object for cancelling service requests
+ * @brief Cancels service requests
  */
 class CORE_API CancellationToken {
  public:
-  /**
-   * @brief Default constructor
-   */
+  /// The alias for the cancellation function.
+  using CancelFuncType = std::function<void()>;
+
   CancellationToken() = default;
 
   /**
-   * @brief Constructor
-   * @param func an operation that should be used to cancel an authentication
-   * request.
+   * @brief Creates the `CancellationToken` instance.
```

Comment:
```suggestion
   * @brief Creates the `CancellationToken` instance.
   *
```
Just to be consistent with the other descriptions, please add an empty line after `@brief`.